layouts/partials/platen/docs/inject/head.html

```diff
@@ -0,0 +1,2 @@
+{{- $platenStyles := resources.Get "platen.scss" | resources.ExecuteAsTemplate "platen.scss" . | resources.ToCSS | resources.Minify | resources.Fingerprint }}
+<link rel="stylesheet" href="{{ $platenStyles.RelPermalink }}" {{ template "integrity" $platenStyles }}>
```

Comment:
Nit: missing trailing newline.
```suggestion
<link rel="stylesheet" href="{{ $platenStyles.RelPermalink }}" {{ template "integrity" $platenStyles }}>

```